### ISSUE TITLE
Git URL incorrect in devshop

### ISSUE THREAD

**Issue by indytechcook** [department-of-veterans-affairs/va.gov-cms#1678]
The git origin url on some PR branches in devshop is setting to `git@github.com:department-of-veterans-affairs/vets-website.git` instead of the va.gov-cms repo.

See http://devshop.cms.va.gov/node/21214 for failing test.

```
aegir@devshop ~/projects/cms/pr1609 (VACMS-1608-health-service-editorial-experience=) $ git remote -v
origin	https://github.com/kevwalsh/va.gov-cms.git (fetch)
origin	git@github.com:department-of-veterans-affairs/vets-website.git (push)
aegir@devshop ~/projects/cms/pr1609 (VACMS-1608-health-service-editorial-experience=) $ git status
On branch VACMS-1608-health-service-editorial-experience
Your branch is up-to-date with 'origin/VACMS-1608-health-service-editorial-experience'.

nothing to commit, working tree clean
aegir@devshop ~/projects/cms/pr1609 (VACMS-1608-health-service-editorial-experience=) $ git remote set-url origin https://github.com/kevwalsh/va.gov-cms.git
aegir@devshop ~/projects/cms/pr1609 (VACMS-1608-health-service-editorial-experience=) $ git status
On branch VACMS-1608-health-service-editorial-experience
Your branch is up-to-date with 'origin/VACMS-1608-health-service-editorial-experience'.

nothing to commit, working tree clean
aegir@devshop ~/projects/cms/pr1609 (VACMS-1608-health-service-editorial-experience=) $ git remot e-v
git: 'remot' is not a git command. See 'git --help'.

The most similar command is
	remote
aegir@devshop ~/projects/cms/pr1609 (VACMS-1608-health-service-editorial-experience=) $ git remote -v
origin	https://github.com/kevwalsh/va.gov-cms.git (fetch)
origin	git@github.com:department-of-veterans-affairs/vets-website.git (push)
aegir@devshop ~/projects/cms/pr1609 (VACMS-1608-health-service-editorial-experience=) $ git remote remove origin
aegir@devshop ~/projects/cms/pr1609 (VACMS-1608-health-service-editorial-experience) $ git remote add origin https://github.com/kevwalsh/va.gov-cms.git
aegir@devshop ~/projects/cms/pr1609 (VACMS-1608-health-service-editorial-experience) $ git remote -v
origin	https://github.com/kevwalsh/va.gov-cms.git (fetch)
origin	https://github.com/kevwalsh/va.gov-cms.git (push)
```

**Comment by ElijahLynn** [comment]
We didn't actually do anything code wise to resolve this right? 

I think this was to do with tasks running out of order, which resolved in one of the sub-issues in #1688. We can mark this as done now I think.

**Comment by indytechcook** [comment]
I agree, it resolved with fixing the number of instances running at the same time.
